app/service/rest.py

```diff
@@ -297,7 +297,7 @@ def update_service(service_id):
         if not fetched_service.restricted:
             _warn_service_users_about_message_limit_changed(service_id, current_data)
     if sms_limit_changed:
-        redis_store.delete(sms_daily_count_cache_key(service_id))
+        delete_daily_sms_fragment_count(service_id)
         redis_store.delete(near_sms_daily_limit_cache_key(service_id))
```

Comment:
`near_sms_daily_limit_cache_key` and `over_sms_daily_limit_cache_key` don't appear to be used anywhere in the API - Are they for future enhancements?

Reply:
We use them to check if we've already sent the service owners the warning / alert emails
https://github.com/cds-snc/notification-api/blob/c8393182334506d1ec0d0648319bed6569021ea4/app/notifications/validators.py#L180-L181
https://github.com/cds-snc/notification-api/blob/c8393182334506d1ec0d0648319bed6569021ea4/app/notifications/validators.py#L202-L203